<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>903</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Unofficial Level Manager for Baba Is You</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="2">
        <widget class="QListView" name="LevelList"/>
       </item>
       <item row="0" column="1">
        <widget class="QFrame" name="frame_2">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="4" column="0">
           <widget class="QLabel" name="SetExportNameLabel">
            <property name="text">
             <string>Set Level Number</string>
            </property>
           </widget>
          </item>
          <item row="11" column="0">
           <widget class="QPushButton" name="SteamPathButton">
            <property name="text">
             <string>Set Steam Path</string>
            </property>
           </widget>
          </item>
          <item row="11" column="1">
           <widget class="QPushButton" name="SavePathButton">
            <property name="text">
             <string>Set Export Path</string>
            </property>
           </widget>
          </item>
          <item row="12" column="0" colspan="2">
           <widget class="QPushButton" name="RefreshButton">
            <property name="text">
             <string>Refresh</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QDoubleSpinBox" name="SetExportNameSpinbox">
            <property name="decimals">
             <number>0</number>
            </property>
            <property name="maximum">
             <double>100000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QPushButton" name="ExportSetNameButton">
            <property name="text">
             <string>Export</string>
            </property>
           </widget>
          </item>
          <item row="10" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="6" column="0" colspan="2">
           <widget class="QPushButton" name="ExportButton">
            <property name="text">
             <string>Export and keep Level Number</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="StatusLabel">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="7" column="0" colspan="2">
           <widget class="QPushButton" name="ImportButton">
            <property name="text">
             <string>Import</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>903</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSet_Path">
   <property name="text">
    <string>Set Steam Path</string>
   </property>
  </action>
  <action name="actionSet_Export_Path">
   <property name="text">
    <string>Set Export Path</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
